<ui version="4.0" >
 <class>transferDialog_q</class>
 <widget class="QDialog" name="transferDialog_q" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>615</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize" >
   <size>
    <width>32767</width>
    <height>32767</height>
   </size>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle" >
   <string>Config transfer options</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QFrame" name="titleFrame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>32767</width>
       <height>32767</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth" >
      <number>1</number>
     </property>
     <property name="midLineWidth" >
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="dialogTitleLine" >
        <property name="text" >
         <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p align="center" style=" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:x-large; font-weight:600;">Export firewall configs to transfer device&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QFrame" name="mainBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" colspan="2" >
       <widget class="QLabel" name="volumeLabel" >
        <property name="text" >
         <string>Select usbstick volume to start firewall config transfer:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QTableWidget" name="transferTable" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="toolTip" >
         <string>Select a portable disk volume to transfer firewall configs</string>
        </property>
        <property name="editTriggers" >
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="tabKeyNavigation" >
         <bool>false</bool>
        </property>
        <property name="showDropIndicator" stdset="0" >
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors" >
         <bool>true</bool>
        </property>
        <property name="selectionMode" >
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior" >
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="textElideMode" >
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="sortingEnabled" >
         <bool>true</bool>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <column>
         <property name="text" >
          <string>Device</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Size</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Mounted</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Mountpoint</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Filesystem</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QTreeWidget" name="fwWorkList" >
        <property name="toolTip" >
         <string>Double click on a firewall in this tree to jump to the corresponding process log entry</string>
        </property>
        <property name="itemsExpandable" >
         <bool>false</bool>
        </property>
        <column>
         <property name="text" >
          <string>Firewall</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Progress</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="2" column="1" >
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QGroupBox" name="detailMCframe_2" >
          <property name="title" >
           <string>Process log</string>
          </property>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="0" column="0" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>176</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1" >
            <widget class="QPushButton" name="saveLogButton" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="text" >
              <string>Save log to file</string>
             </property>
             <property name="icon" >
              <iconset resource="MainRes.qrc" >
               <normaloff>:/Icons/save_25.png</normaloff>:/Icons/save_25.png</iconset>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2" >
            <widget class="QTextEdit" name="procLogDisplay" >
             <property name="readOnly" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QFrame" name="buttonsFrame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>32767</width>
       <height>32767</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="1" >
       <widget class="QPushButton" name="transferButton" >
        <property name="toolTip" >
         <string>Start the export!</string>
        </property>
        <property name="text" >
         <string>Transfer</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="cancelButton" >
        <property name="text" >
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>transferButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="MainRes.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>transferButton</sender>
   <signal>clicked()</signal>
   <receiver>transferDialog_q</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>transferDialog_q</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>transferTable</sender>
   <signal>cellClicked(int,int)</signal>
   <receiver>transferDialog_q</receiver>
   <slot>selected()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>277</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel" >
     <x>277</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveLogButton</sender>
   <signal>clicked()</signal>
   <receiver>transferDialog_q</receiver>
   <slot>saveLog()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>483</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel" >
     <x>277</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fwWorkList</sender>
   <signal>itemActivated(QTreeWidgetItem*,int)</signal>
   <receiver>transferDialog_q</receiver>
   <slot>findFirewallInLog(QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>143</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel" >
     <x>277</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
